verification/helpers.go

```diff
@@ -1,13 +1,51 @@
 package verification
 
 import (
+	"encoding/json"
 	"fmt"
+	"os"
+	"path/filepath"
 	"regexp"
 	"strings"
 
 	ldapv3 "github.com/go-ldap/ldap/v3"
 )
 
+func loadPolicyDocument(policyDocumentPath string) (*PolicyDocument, error) {
+	var policyDocument *PolicyDocument = &PolicyDocument{}
+	jsonFile, err := os.Open(policyDocumentPath)
+	if err != nil {
+		return nil, err
+	}
+	defer jsonFile.Close()
+	err = json.NewDecoder(jsonFile).Decode(policyDocument)
+	if err != nil {
+		return nil, err
+	}
+	return policyDocument, nil
+}
+
+func loadX509TrustStores(policyDocument *PolicyDocument, trustStoreBasePath string) (map[string]*X509TrustStore, error) {
+	var result = make(map[string]*X509TrustStore)
+	for _, trustPolicy := range policyDocument.TrustPolicies {
+		for _, trustStore := range trustPolicy.TrustStores {
+			if result[trustStore] != nil {
+				// we loaded this trust store already
+				continue
+			}
+			i := strings.Index(trustStore, ":")
+			prefix := trustStore[:i]
+			name := trustStore[i+1:]
```

Comment:
try to use [strings.Cut](https://pkg.go.dev/strings#Cut) as `i` might be `-1`.

Reply:
It can't be `-1` because we are validating it at https://github.com/notaryproject/notation-go/blob/main/verification/policy.go#L129